pages/profiles/[id].tsx

```diff
@@ -0,0 +1,195 @@
+import Head from "next/head";
+import Navbar from "../../components/navbar";
+import style from "../../styles/profile.module.scss";
+import profile from "../../public/profile.jpg";
+import noimage from "../../public/no-image-icon-6.png";
+import plus_sign from "../../public/plus-sign.png";
+import Image from "next/image";
+import { StaticImageData } from "next/image";
+import Link from "next/link";
+import { ChangeEvent, SetStateAction, useState, Dispatch, useEffect, useRef, Component } from "react";
+import { IPublicClientApplication, AccountInfo } from '@azure/msal-browser';
+import { useMsal } from '@azure/msal-react';
+import { loginRequest } from "../../azureAuth.config";
+import { callMsGraph } from "../../azureGraph.config";
+import { useIsAuthenticated } from '@azure/msal-react';
+import { updateDescription, findUserByMsftProvider, findUserById, UserData, EMPTY_USER_DATA } from "../../clients/user_service";
+import { find_projects_by_user_id, ProjectData } from "../../clients/project_service";
+import { Button, CircularProgress, Skeleton } from "@mui/material";
+import Footer from "../../components/footer";
+import { getCurrentUserId } from "../../clients/azure_client";
+import { useRouter } from "next/router";
+
+const full_name = "John Doe";
+const semester_batch = "Semester 5, Batch 2025";
+const current_description = "Hello, my name is John Doe";
+
+
+function handleChangeDescription(event: ChangeEvent<HTMLTextAreaElement>, state: string, setter: Dispatch<SetStateAction<string>>) {
+    const new_description = event.target.value;
+    setter(oldvalue => new_description);
+}
+
+type ProfileDescriptionProps = {
+    user_id: string,
+    description: string,
+}
+
+function ProfileDescription(profileDescriptionProps: ProfileDescriptionProps) {
+    const { user_id, description: propsDescription } = profileDescriptionProps;
+    const [editting, setEditting] = useState(false);
```

Comment:
@Bumskee This editting is also not used anymore. So it's fine, you can remove it.

Reply:
already removed it :D